CONTRIBUTING.md

```diff
@@ -89,3 +89,83 @@ Reference [pre-commit's installation instructions](https://pre-commit.com/#insta
 ```
 
 See `.pre-commit-config.yaml` for a list of configured linters and fixers.
+
+## Development Environment
+
+### Running Docker Containers
+
+When `docker-compose up` is run from the `deploy/codex/` directory, several Docker containers are started. These are the connected components of the Codex application.
+
+You can view them by using the command `docker ps -a` and see something like this: 
```

Comment:
`docker ps -a` includes all the running containers in your system, which is useful.  But to just look at the relevant containers from this particular docker-compose, you can do `docker-compose ps`, for example:

```
(py3) karen@yukihira:~/src/wildme/houston/deploy/codex$ docker-compose ps
     Name                    Command                  State                      Ports               
-----------------------------------------------------------------------------------------------------
codex_acm_1       /docker-entrypoint.sh wait ...   Up             0.0.0.0:82->5000/tcp               
codex_db_1        docker-entrypoint.sh postgres    Up             5432/tcp                           
codex_edm_1       /usr/local/tomcat/bin/cata ...   Up             0.0.0.0:81->8080/tcp               
codex_gitlab_1    /assets/wrapper                  Up (healthy)   22/tcp, 443/tcp, 0.0.0.0:85->80/tcp
codex_houston_1   /docker-entrypoint.sh wait ...   Up             0.0.0.0:83->5000/tcp               
codex_redis_1     docker-entrypoint.sh redis ...   Up             6379/tcp
```

Reply:
Cool. Certainly learning a few things about docker-compose from these comments, thank you.

Reply:
Ah, one disadvantage is that you need to be in the deploy/codex/ directory while docker commands can be used anywhere. NBD though, lets settle on the docker-compose commands.